src/main/java/com/axis/eiffel/gerrit/herald/RabbitMQ/Sender.java

```diff
@@ -54,9 +54,13 @@ private Channel createChannel() throws IOException {
      * @param type Exchange type
      * @throws IOException
      */
-    public void setExchange(String exchangeName, String type) throws IOException {
+    public void setExchange(String exchangeName, String type, boolean activeDeclare) throws IOException {
```

Comment:
Add the new boolean to the doc string above.

Reply:
Doh, that was sloppy. Done.